### DISCUSSION TITLE
如何修改electron 的标题名称？

### DISCUSSION THREAD

**Discussion by hanbingzi** [opensumi/core#3106] [Q&A] [answered]
1. 如何修改electron 的标题名称，如下图？
2. 注册程序标题的下拉和注册菜单是一个方法吗？
<img width="488" alt="image" src="https://github.com/opensumi/core/assets/20720644/51a372a8-0d38-421f-8049-87a6cd96e4e0">

**Comment by bytemain** [comment, marked as answer]
这个标题名称应该是 Electron 的应用名称，可以参考 Electron 的官方文档。

https://www.electronjs.org/docs/latest/api/app#appname

是需要你配置在 package.json 中和 electron-builder 等打包工具中的。
